ansible/playbook.yml

```diff
@@ -147,3 +147,11 @@
         state: restarted
       tags:
         - deploy
+    - name: send notification on slack
+      slack:
+        token: "{{ slack.notifications.token }}"
```

Comment:
I think you should be okay to just use the `slack.oauth_token` here, instead of something _else_. I need to see who that is tied to, I think it's mine 🙊

Reply:
Great! In that case I've removed it